### ISSUE TITLE
simple-line-icons are not working

### ISSUE THREAD

**Issue by sabbir25112** [kevinchappell/formBuilder#701]
### Description:
In my project, I use simple-line-icons for the sidebar. Now I need to use formbuilder, but when I include form-builder.min.js it maybe overrides simple-line-icons. So icons are broken. I can't find any solution. 

### Screenshot - *(optional)*
![image](https://user-images.githubusercontent.com/17101635/35766727-a138b06c-0907-11e8-8e20-b226620ebc48.png)

**Comment by arrgh11** [comment]
I had this same issue. Simple-line-icons uses ".icon-" as their class prefix, and so does FormBuilder, which is why it's overriding your other icons. So how I got around it was downloading and modifying the "simple-line-icons" css file and changing all the icon names to include ".simple" as the new prefix and once I relinked the new css, changed my <i></i> tags to the new ".simple-icon-", etc.,  and that fixed it! Hope this helps.
